types/features/site-templates.interfaces.ts

```diff
@@ -23,10 +23,21 @@
 // POSSIBILITY OF SUCH DAMAGE.
 
 export type SiteTemplatesMetadata = {
-    screenshot_dimensions?: string;
+    screenshot_dimensions: {
```

Comment:
we've lost the optionality there? Is that intentional? (I would imagine so, but...)

Reply:
To be honest I don't assume third parties can use the dimensions for anything useful (we dont publish the image urls) so in my opinion we might as well remove meta from the endpoint.

As far as I'm concerned the only place the metadata is used is in the puppeteer script